### DISCUSSION TITLE
Country area issues with India

### DISCUSSION THREAD

**Discussion by kyssaFarms** [saleor/saleor#9226] [Q&A]
Hi,
For both warehouse setting and site setting I choose the country as India and whatever random variations of letters i've tried to put together the country area always shows invalid input. Is there a list of country area codes I can refer to or is this a possible bug?

**Comment by moontaincow** [comment]
> Hi, For both warehouse setting and site setting I choose the country as India and whatever random variations of letters i've tried to put together the country area always shows invalid input. Is there a list of country area codes I can refer to or is this a possible bug?

I was worried after reading this and went to create a warehouse in India, but it seemed to work fine for me. I set the city to Ernakulam, searched for a valid zip code, with country area set to Kerala.
